packages/formation/sass/full.scss

```diff
@@ -32,3 +32,4 @@ $image-path: '~uswds/src/img';
 @import 'modules/m-promo-banner';
 @import 'modules/m-responsive-tables';
 @import 'modules/m-back-to-top';
+@import 'modules/m-action-link';
```

Comment:
There's another file that we might want to import this into. I'm honestly not sure what the difference is, but the pattern is there, so :shrug:.

Reply:
I looked at `core.scss`, but it didn't import the modules. What other file were you thinking of?

Reply:
This one: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/site/site.scss

Reply:
Ah, good call. :+1: Thanks!